backend/src/modules/Announcements/service/Announcement.service.spec.js

```diff
@@ -0,0 +1,63 @@
+import { InMemoryAnnouncementRepository } from '../repository/inMemory/InMemoryAnnouncement.repository.js';
+import { AnnouncementsService } from './Announcements.service.js';
+
+describe('AnnouncementService |', () => {
+  const correctAnnouncementData = {
+    title: 'Brak wody',
+    boardName: 'Sobótka',
+    content: 'Nie będzie wody tego i tego dnia, tra la la.',
+  };
+
+  const announcementDataWithoutContent = {
+    title: 'Brak wody',
+    boardName: 'Sobótka',
+  };
+
+  const announcementDataWithToShortTitle = {
+    title: 'lol',
+    boardName: 'Sobótka',
+    content: 'Nie będzie wody tego i tego dnia, tra la la.',
+  };
+
+  const boardId = '8a4a5d2d-af04-4f3a-87d1-ef8130f8bbb1';
+
+  test('when correct announcement is added, then board announcements is returned', async () => {
+    //Given
+    const inMemoryAnnouncementRepository = new InMemoryAnnouncementRepository();
+    const announcementService = new AnnouncementsService(inMemoryAnnouncementRepository);
+
+    //When
+    const boardAnnouncements = await announcementService.addAnnouncement(
+      correctAnnouncementData,
+      boardId,
+    );
+
+    //Then
+    expect(Array.isArray(boardAnnouncements)).toBeTruthy();
+    expect(boardAnnouncements[0]).toHaveProperty('id');
+    expect(boardAnnouncements[0]).toHaveProperty('title');
+    expect(boardAnnouncements[0]).toHaveProperty('boardName');
+    expect(boardAnnouncements[0]).toHaveProperty('content');
+    expect(boardAnnouncements[0]).toHaveProperty('commentsIds');
+    expect(boardAnnouncements[0]).toHaveProperty('date');
```

Comment:
A coś w tym stylu?
https://jestjs.io/docs/expect#toequalvalue

```suggestion
    //Then
    const expectedBoardAnnouncment = {
        id: "aweweqsasa"
        title: "asas"
        boardName: "dsfs"
    }
    expect(Array.isArray(boardAnnouncements)).toBeTruthy();
    expect(boardAnnouncements[0]).toEqual((expectedBoardAnnouncment);
```